tutorials/ngsi_v2/e6_timeseries_data/e6_timeseries_data_solution.py

```diff
@@ -282,45 +293,47 @@ def on_measurement(client, obj, msg):
     # drop unnecessary index levels
     history_weather_station = history_weather_station.droplevel(
         level=("entityId", "entityType"), axis=1)
-    history_weather_station['simtime'] = pd.to_numeric(
-        history_weather_station['simtime'], downcast="float")
+    history_weather_station['sim_time'] = pd.to_numeric(
+        history_weather_station['sim_time'], downcast="float")
     history_weather_station['temperature'] = pd.to_numeric(
         history_weather_station['temperature'], downcast="float")
-    # ToDo: plot the results
+    # ToDo: Plot the results.
     fig, ax = plt.subplots()
-    ax.plot(history_weather_station['simtime'],
+    ax.plot(history_weather_station['sim_time']/60,
```

Comment:
Here and in the plots below again the x-axis has been changed, but it is probably not working correctly, see the comment on plotting in the previous exercise.